src/Datadog.Trace/FrameworkDescription.cs

```diff
@@ -0,0 +1,264 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using Datadog.Trace.Logging;
+using Microsoft.Win32;
+
+namespace Datadog.Trace
+{
+    internal class FrameworkDescription
+    {
+        private static readonly ILog Log = LogProvider.GetCurrentClassLogger();
+
+        private static readonly Assembly RootAssembly = typeof(object).GetTypeInfo().Assembly;
```

Comment:
nit: Silly question, but is there anything wrong with doing `typeof(object).Assembly`? The `GetTypeInfo()` call seems unnecessary at first glance.

Reply:
Oops. I got used to doing this because it was required in .NET Core before 2.0.

From https://docs.microsoft.com/en-us/dotnet/api/system.type.assembly
> If the `Type.Assembly` property is not available on a particular .NET implementation, such as .NET Core or the Universal Windows Platform, use the `TypeInfo.Assembly` property instead.

We don't need the extra method call anymore so I'll remove it.

Reply:
Removed in ab878a61eff037cddd6e179eaeae70e7be33af1c.